CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The sections should follow the order `Added`, `Changed`, `Fixed`, and `Removed`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+- On systems using Fontconfig backend newly added fonts not visible upon `Rasterizer` recreation.
```

Comment:
This isn't really accurate, is it? If things aren't visible upon `Rasterizer` (re)creation, this wouldn't help, since this only applies ***after*** the rasterizer has been created. So it applies to *loading* new fonts after rasterizer creation.

I think the following is a better description:

```
- Fontconfig not checking for new fonts installed after `Rasterizer` creation when calling `load_font`
```